bbp/nse/placement-algorithm/default.nix

```diff
@@ -24,12 +24,12 @@ in
 
 stdenv.mkDerivation rec {
   name = "placement-algorithm-${version}";
-  version = "2018.07.12-${stdenv.lib.substring 0 6 src.rev}";
```

Comment:
why do you put the date in the version ?

Reply:
Because hash does not tell me much? ^^
That's one of versioning conventions Adrien was using, as far as I recall; and it helps at least to have an idea how old the module version is.